dist/app.js

```diff
@@ -0,0 +1,32 @@
+var modeEL = document.getElementById('mode');
+var valueEL = document.getElementById('value');
+var numButtons = document.getElementById('number-button');
```

Comment:
Refer to your `index.html`. There are many `.number-button` elements so this should be `getElementsByClassName`.